cli/cli.ts

```diff
@@ -1525,16 +1540,20 @@ function buildSemanticUIAsync() {
 }
 
 function updateDefaultProjects(cfg: pxt.TargetBundle) {
-    let templatesRoot = path.join("libs", "templates");
-    let defaultProjects = [
+    let defaultTemplates = [
         "blocksprj",
         "tsprj"
     ];
 
-    defaultProjects.forEach((p) => {
-        let projectPath = path.join(templatesRoot, p);
+    getTemplates().forEach((templatePath) => {
+        if (defaultTemplates.indexOf(path.basename(templatePath)) === -1) {
```

Comment:
Isn't this an overkill? If we have more templates in the future, we'll deal with that.